tests/MySphinxTestCase.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
If such file extracted it should be placed under 'tests/data' folder, otherwise it produces confusion.